terraform/aws/modules/network/main.tf

```diff
@@ -157,24 +157,153 @@ resource "aws_lb" "km_lb" {
   name            = "km-lb-${var.environment}"
   subnets         = aws_subnet.km_public_subnet.*.id
   security_groups = [aws_security_group.km_alb_sg.id]
+
+  access_logs {
+    bucket  = "<s3_bucket_name>"
+    enabled = true
+  }
 }
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure one target group is configured to listen on HTTPS for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one target group listening on HTTPS ensures end-to-end encryption is not enabled. This could impact the confidentiality of data in transit.
 
**Remediation**
Elastic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: tenablecs-